### DISCUSSION TITLE
Deployment in azure kubernetes

### DISCUSSION THREAD

**Discussion by johnsnow2022** [anasty17/mirror-leech-telegram-bot#1401] [Q&A]
I am using azure cloud services when I try to deploy the bot using azure kubernetes (deployment center)
I am getting response for bot but the response is repeated and if I leech a file it is doing the process twice 
Is there any possibile solution

**Comment by anasty17** [comment]
U deployed the bot twice! I guess. Seem u r newbie. Solve it by ur self or delete the instance

**Reply by johnsnow2022** [reply]
Definitely not i try plenty number of times and I am shure that the bot isn't deployed twice 
I tried by deleting all the kubernetes cluster
But if I deploy the bot in vps the bot work 100 % fine

**Reply by anasty17** [reply]
Then sorry idk what azure kubernetes is. I leave this discussion so if anyone else know about

**Reply by johnsnow2022** [reply]
Anyways thanks for support 🙂
